doc/classes/Transform2D.xml

```diff
@@ -78,6 +78,13 @@
 				This method does not account for translation (the origin vector).
 			</description>
 		</method>
+		<method name="determinant" qualifiers="const">
+			<return type="float" />
+			<description>
+				Returns the determinant of the basis matrix. If the basis is uniformly scaled, its determinant is the square of the scale.
```

Comment:
Not sure if the second sentence is a vital information. :thinking: But if anything I'd reword it something like:
```suggestion
				Returns the determinant of the basis matrix. If the basis is uniformly scaled, then its determinant equals the square of the scale factor.
```
- "then" I think it's tend to be used explicitely within the docs, seen it many times like that. Not sure about the change.
- "equals" seems more fitting to me.
- "scale factor" `scale` is still kinda a two component vector, I find "scale factor" clearer (it refers to the scalar).